.github/workflows/deploy_android_canary.yml

```
@@ -72,5 +67,4 @@ jobs:
           SIGNING_KEY_ALIAS: ${{ secrets.SIGNING_KEY_ALIAS }}
           SIGNING_KEY_PASSWORD: ${{ secrets.SIGNING_KEY_PASSWORD }}
           PACKAGE_NAME: me.tinykitten.trainlcd.dev
-          NODE_ENV: production
-          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          NODE_ENV: production
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)